src/pdf_beamtime/src/simple_server.cpp

```diff
@@ -0,0 +1,182 @@
+#include <pdf_beamtime/simple_server.hpp>
+
+using moveit::planning_interface::MoveGroupInterface;
+
+SimpleServer::SimpleServer(
+  const std::string & move_group_name = "ur_manipulator",
+  const rclcpp::NodeOptions & options = rclcpp::NodeOptions())
+: node_(std::make_shared<rclcpp::Node>("simple_server", options))
+{
+  // Add the obstacles
+  this->planning_scene_interface = new moveit::planning_interface::PlanningSceneInterface();
+  this->planning_scene_interface->applyCollisionObjects(create_env());
+
+  this->move_group_interface_ = new MoveGroupInterface(this->node_, move_group_name);
```

Comment:
Make this using an initializer list, similar to node_ https://github.com/maffettone/erobs/blob/07e61ea4365613a071ecead56364827a07ff505b/src/hello_moveit/src/pick_place_repeat_server.cpp#L40-L41

Reply:
moved to the initializer list. 
https://github.com/ChandimaFernando/erobs/blob/9f696b7d2d205fd2038f6a764db896805352b3fb/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L10